caddyhttp/httpserver/roller.go

```diff
@@ -19,14 +20,30 @@ type LogRoller struct {
 }
 
 // GetLogWriter returns an io.Writer that writes to a rolling logger.
+// This should be called only from the main goroutine (like during
```

Comment:
I guess I could, I just didn't bother to since this is only called during the setup which happens in the main goroutine. Maybe if the need arises we can make it thread-safe?